docs/source/levels/intermediate_level_14.rst

```diff
@@ -0,0 +1,43 @@
+#################################
+Level 14: Run on on-prem clusters
+#################################
+
+In this level you'll learn to run on onprem clusters.
+
+----
+
+.. raw:: html
+
+    <div class="display-card-container">
+        <div class="row">
+
+.. Add callout items below this line
+
+.. displayitem::
+   :header: Run on an on-prem cluster
+   :description: Learn to train models on a general compute cluster.
+   :col_css: col-md-4
+   :button_link: ../clouds/cluster_intermediate_1.html
+   :height: 150
+   :tag: intermediate
+
+.. displayitem::
+   :header: Run on a SLURM cluster
+   :description: Run models on a SLURM-managed cluster
+   :col_css: col-md-4
+   :button_link: ../clouds/cluster_advanced.html
+   :height: 150
+   :tag: intermediate
+
+.. displayitem::
+   :header: Run with Torch Distributed
+   :description: Run models on a cluster with torch distributed.
```

Comment:
Maybe "torch distributed" could be shown as "`torch.distributed`" so people know it's a package and not a description.